misc/alert_rules.json

```diff
@@ -276,10 +276,6 @@
   "rule": "%devices.os = \"nxos\" && %sensors.sensor_type = \"cefcFanTrayOperStatus\" && %sensors.sensor_current = \"[3-4]\"",
   "name": "Cisco NX-OS device has a bad fan"
   },
-  {
```

Comment:
Any reason this was removed, it should be valid still as it checks that the previous state was passive but is not active.

Reply:
The rule was tested in deep yesterday and no way to match it and I didn't find a way to make it work. Don't understand why but since this is an important one I prefer to remove it to prevent users from using it if it's not working.